examples/daal/cpp_sycl/makefile_lnx

```diff
@@ -87,8 +84,7 @@ else
 endif
 
 ifeq ($(threading),sequential)
-    DAAL_LIB_T := $(DAAL_LIB_EXT),-lonedal_sequential
-    EXT_LIB += $(addprefix -L,$(TBB_PATH)) -ltbb -ltbbmalloc
+    $(error Sequential version is not supported)
```

Comment:
The message need to be aligned with `makefile_win`

Reply:
fixed